app/controllers/ops_controller/diagnostics.rb

```diff
@@ -350,6 +347,17 @@ def diagnostics_server_list
 
   private ############################
 
+  def fetch_journal_log(filter_params)
+    require "systemd/journal"
+    journal = Systemd::Journal.new
+    begin
+      journal.filter(filter_params)
+      journal.map(&:message)
+    ensure
+      journal.close
+    end
+  end
```

Comment:
We may want this to be a little smarter to know if appliance then journald else from files, because local dev will still have files.  Devs should know how to get logs directly, but IMO it would be weird, particularly in test, that this would be empty locally.

Reply:
If we want to handle log files on dev we could have a `fetch_log` method which handled `if systemd => journal elsif !container => files` or something like that

Reply:
Crap i didn't read the TODO hahaha